elastic/kibana
# [SecuritySolution] Update configs to apply tags in descriptions

## Summary

Tags not working after https://github.com/elastic/kibana/pull/183611

**To test security_solution_endpoint:**
```
cd x-pack/test/security_solution_api_integration

node ../../../scripts/functional_tests_server.js --config ./test_suites/security_solution_endpoint/configs/serverless.endpoint.config.ts
```
Once the server is launched (you might need Docker to run the serverless tests), open another terminal, go to the same path, and execute the command appears in the original:


The command should look like: 
```
node ../../../scripts/functional_test_runner --config=test_suites/security_solution_endpoint/configs/serverless.endpoint.config.ts
```


**To test security_solution_endpoint:**

```
cd x-pack/test/security_solution_endpoint_api_int

node ../../../scripts/functional_tests_server.js --config ./test_suites/security_solution_endpoint_api_int/configs/config.ts

```


```

open a new terminal and go to the same folder:

node ../../../scripts/functional_test_runner --config=test_suites/security_solution_endpoint_api_int/configs/config.ts
```
